bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusSlave.java

```diff
@@ -198,10 +200,9 @@ private void setCoil(Command command, ModbusBindingConfig config)
      * @throws ModbusException ModbusIOException on IO errors, ModbusSlaveException with protocol level exceptions
      * @throws ModbusUnexpectedTransactionIdException when response transaction id does not match the request
      */
-    protected void setRegister(Command command, ModbusBindingConfig config)
+    protected void setRegister(Command command, ModbusBindingConfig config, int readIndex, int writeRegister)

```

Comment:
Sure, I can change it as described in the source-code snippeds.

Do you mean by implementing the Rollershutter by rules and leave the code as it is?